ui/src/main/webapp/index.js

```diff
@@ -1,44 +1,32 @@
 import inject from 'react-tap-event-plugin'
 import React from 'react'
-import { render } from 'react-dom'
-import { Router, createMemoryHistory, hashHistory, RouterContext, match } from 'react-router'
+import ReactDOM from 'react-dom'
```

Comment:
The `{ render }` variable introduced would clobber the new render definition in the file. We are still importing them same amount of code. It's just not being de-structured like it was before.